includes/wikia/Wikia.php

```diff
@@ -310,15 +312,15 @@ static public function fixDomainName( $name, $language = false, $type = false )
 					case "answers":
 						$domains = self::getAnswersDomains();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)